charts/microgateway/README.md

```diff
@@ -95,6 +95,7 @@ The following table lists configuration parameters of the Airlock Microgateway c
 | config.simple.backend.port | int | `8080` | Backend port |
 | config.simple.backend.protocol | string | `"http"` | Backend protocol |
 | config.simple.mapping.denyRules.enabled | bool | `true` | Enable all Deny rules. |
+| config.simple.mapping.denyRules.exceptions | list | `[]` | Specification of deny rule exceptions. |
```

Comment:
Change "Specification of deny rule exceptions." to "Deny rule exceptions" or "Deny rule exceptions configuration."

In my opinion it is not a specification, but a configuration or setting, ...